### PR TITLE
Added smooth scrooling to all pages

### PR DESCRIPTION
**Pull request by AnshuPathak-88825**

### CODE REVIEW

**Comment by AnshuPathak-88825** [comment]
[Screencast from 27-08-22 09:47:20 PM IST.webm](https://user-images.githubusercontent.com/78332454/187038769-823155e5-8ae2-4bae-9af7-18376d03c5c0.webm)
It is working perfectly

**Review by myhacktech — APPROVED** [review]

**Comment by AliReza1083** [comment]
Great Job Brother

File: js/scrollsmooth.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const scroll_button=document.getElementsByClassName(".smooth-scroll");
```
Comment (AliReza1083) [inline js/scrollsmooth.js:1]: If you used `querySelector`, That'd be great But btw Great JOB.

File: js/scrollsmooth.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+for(let i of scroll_button)
+{
+    i.addEventListener('click',function(e){
+        e.preventDefault();
+        const target=this.getAttribute('href');
+        const target_position=document.querySelector(target);
+        console.log(target_position)
+        target_position.scrollIntoView({scrollTop:target.scrollTop,behavior:'smooth'}); 
+
```
Comment (AliReza1083) [inline js/scrollsmooth.js:10]: It'd be better if you remove the extra lines...

File: js/scrollsmooth.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+const scroll_button=document.getElementsByClassName(".smooth-scroll");
+for(let i of scroll_button)
+{
+    i.addEventListener('click',function(e){
+        e.preventDefault();
+        const target=this.getAttribute('href');
```
Comment (AliReza1083) [inline js/scrollsmooth.js:6]: For writing a clean code, it is better to add space before and after equal mark.
For exemple: 
``` diff
- const target=this.getAttribute('href');
+ const target = this.getAttribute('href');
```

**Review by AliReza1083 — COMMENTED** [review]
If you follow these 3 things, Your code will look Awesome.
Wish you all the best Brother.

File: js/scrollsmooth.js (hunk 3 above)
Comment (AnshuPathak-88825) [reply js/scrollsmooth.js:6]: thanks brother I won't do it again

**Comment by Susmita-Dey** [comment]
@AliReza1083 Tag me when @AnshuPathak-88825 have solved the issues

**Comment by AnshuPathak-88825** [comment]
@AliReza1083 and @Susmita-Dey pls review changes and merge it

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by Susmita-Dey** [comment]
@AnshuPathak-88825 Fix the merge conflicts.

**Comment by AnshuPathak-88825** [comment]
fixed merge conflicts

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Reusable

**Comment by Susmita-Dey** [comment]
@AnshuPathak-88825 Issue no?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

**Comment by AnshuPathak-88825** [comment]
#230 issue number

**Review by Susmita-Dey — APPROVED** [review]
LGTM. Good job

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**